src/Microsoft.SqlTools.ServiceLayer/LanguageServices/Completion/SqlCompletionItem.cs

```diff
@@ -19,9 +19,9 @@ namespace Microsoft.SqlTools.ServiceLayer.LanguageServices.Completion
     public class SqlCompletionItem
     {
         private static Regex ValidSqlNameRegex = new Regex(@"^[\p{L}_@][\p{L}\p{N}@$#_]{0,127}$");
-        private static DelimitedIdentifier BracketeIidentifiers = new DelimitedIdentifier { Start = "[", End = "]"};
+        private static DelimitedIdentifier BracketeIdentifiers = new DelimitedIdentifier { Start = "[", End = "]"};
```

Comment:
looks like you're still missing a 'd' between 'e' and 'I' if I'm not mistaken :)